### ISSUE TITLE
version 2.0.2 on Webkit not working

### ISSUE THREAD

**Issue by arosenhagen** [FriendsOfShopware/FroshLazySizes#20]
### PHP Version

8.2

### Shopware Version

6.5.6.0

### Actual behaviour

On Webkit (Safari/iOS) version 2.0.2 doesn't display images (blanks).
Maybe related to `Use object-fit given into thumbnail utility`

### Expected behaviour

show all images

### Steps to Reproduce?

Use any webkit based browser

**Comment by pascaltippelt** [comment]
I can confirm, version 1.0.3 does not work on webkit (iphone) too.

**Comment by pumpi** [comment]
I have tested it and can reproduce it in Safari; other WebKit browsers work well. I have moved the product-image fix to Twig, which fixes the Safari problem.

**Comment by d0p3fish** [comment]
Confirmation on iPhone and iMac:

    Shopware 6.5.6.0
    PHP 8.1
    iMac: Safari 17.0
    iPhone: Safari

When attempting to disable the 'object-fit' property for images in the Safari inspector, I encounter a problem. It causes everything on the webpage, as well as the inspector itself, to shift around. The end result is that 'object-fit' remains activated, but the images are now visible.

**Comment by jonas-sfx** [comment]
Confirmation on an iPhone with Safari.
UserAgent: "Mozilla/5.0 (iPhone; CPU iPhone OS 12_5_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Mobile/15E148 Safari/604.1"

Product-Listings are without all the lazy-loaded images.

On Version 1.0.2 the img-Tag was:

```html
<img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
   data-src="https://www.domain.de/media/3e/da/b7/1675343822/IMG12345.jpg"
   data-srcset="https://www.domain.de/media/3e/da/b7/1675343822/IMG12345.jpg 801w, https://www.domain.de/thumbnail/3e/da/b7/1675343822/IMG12345_800x800.jpeg 800w, https://www.domain.de/thumbnail/3e/da/b7/1675343822/IMG12345_400x400.jpeg 400w"
   data-sizes="auto"
   data-aspectratio="1500/2250"
   data-parent-fit="contain"
   class="frosh-proc lazyload product-image is-standard"
   alt="Very nice Product"
   title="Very nice Product"
   loading="lazy"
   style="aspect-ratio:1500/2250;"
   />
   ```
   
 with version 1.0.3 it is:
```html
<img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
   data-src="https://www.domain.de/media/3e/da/b7/1675343822/IMG12345.jpg"
   data-srcset="https://www.domain.de/media/3e/da/b7/1675343822/IMG12345.jpg 801w, https://www.domain.de/thumbnail/3e/da/b7/1675343822/IMG12345_800x800.jpeg 800w, https://www.domain.de/thumbnail/3e/da/b7/1675343822/IMG12345_400x400.jpeg 400w"
   data-sizes="auto"
   data-aspectratio="1"
   class="frosh-proc lazyload product-image is-standard"
   alt="Very nice Product"
   title="Very nice Product"
   loading="lazy"
   style="aspect-ratio:0.66666666666667;"
   />
   ```
 
The Image itself did not change. So i guess it must be something with the aspect-ratio-numbers (or the parent-fit?)

Going back to Version 1.0.2. solved it.
Now I am looking forward to Version 1.0.4 :)

**Comment by jonas-sfx** [comment]
Additional Confirmation on a Desktop-Safari:
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.3 Safari/605.1.15

**Comment by pumpi** [comment]
We don't need any confirmation we already fix it #21 but not merge it at this time, please wait for merge 🙈

**Comment by tinect** [comment]
I'll try to release it tomorrow for 6.4 and 6.5👍
